tests/functional/export_v2/cram/minimal.t

```diff
@@ -16,7 +15,31 @@ Minimal export
   Validation of 'minimal.json' succeeded.
   
 
-  $ python3 "$TESTDIR/../../../../scripts/diff_jsons.py"  "$TESTDIR/../data/minimal.json" minimal.json \
+The above minimal.json takes divergence from the newick file. This converts newick divergences of (e.g.) '1' to `1.0`
+because BioPython uses floats (which is perfectly reasonable). Remove the decimal to diff the JSON.
+(Note that Auspice won't behave any differently)
+  $ sed 's/\.0//' minimal.json > minimal.no-decimal.json
```

Comment:
Could you use `--significant-digits 0` in the call to `diff_jsons.py` below to achieve this same effect without additional copies of the JSONs?

Reply:
Oh cool, I forgot this was an option!

Unfortunately in the case of 0 significant digits we get an associated int vs float change (maybe? It doesn't look like it's removing the `.0` at all, but I presume deepdiff is showing the u-modified input values here)

```console
{'old_type': <class 'int'>, 'new_type': <class 'float'>, 'old_value': 1, 'new_value': 1.0},
```

Reply:
Ah, I'm sorry that wasn't just a one-line fix, @jameshadfield! [DeepDiff supports ignoring type changes for this scenario](https://zepworks.com/deepdiff/current/ignore_types_or_values.html). If you're open to it, I can push a commit that adds a flag for the diff JSONs script to `--ignore-numeric-type-changes`. I bet this would be handy for other tests, too...

Reply:
I added this functionality in 57bdf28f456ea3cd2669294596f90cd4e89018fc.

Reply:
Although this reminded me that [there is a deep diff CLI](https://github.com/nextstrain/augur/issues/1016) that we could eventually switch to... 🤦🏻